modules/ob-defining-the-output.adoc

```diff
@@ -0,0 +1,92 @@
+// This module is included in the following assembly:
+//
+// builds/configuring-openshift-builds.adoc
+
+:_content-type: CONCEPT
+[id="ob-defining-the-output_{context}"]
+= Defining the output
+
+In your `Build` CR, you can specify an output location to push the image. When using an external private registry as your output location, you must specify a secret to access the image. You can also specify the annotations and labels for the output image. 
+
+[NOTE]
+====
+When you specify annotations or labels, the output image will get pushed twice. The first push comes from the build strategy and the second push changes the image configuration to add the annotations and labels. 
+====
+
+The following example defines a public registry where the image will be pushed:
+
+[source,yaml]
```

Comment:
Just a suggestion, but you can also include YAML files in snippets. I just got the suggestion to do this myself a few weeks back, so thought I'd pass it along: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#embedding-a-local-yaml-file

Reply:
@skrthomas, as far as i know, creating a snippet file is useful only if you want to mention it in mutiple modules or at multiple instances within a documentation. All the config examples in this PR are distinct and different and would not reused anywhere. This is why i do not see any use of moving the yaml examples into a snippet file.